libs/membership/website/src/lib/invoice-list/invoice-list-container.tsx

```diff
@@ -81,7 +76,11 @@ export function InvoiceListContainer({
         <StripeElement clientSecret={stripeClientSecret}>
           <StripePayment
             onClose={success => {
-              window.location.href = success ? successURL : failureURL
+              if (stripeMemberPlan) {
+                window.location.href = success
+                  ? stripeMemberPlan.successPage?.url ?? ''
```

Comment:
I would probably redirect to startpage / or /profile by default instead of staying on the /mitmachen page.
If success page is missing, users would probably buy multiple subscriptions.

Reply:
Thats why we have the warnings when you try to create a new subscription that you already have.
I generally want to go away from hardcoded pages, because then I have to pass it down again as they could be different from media to media (for example the URL being in German, not mitmachen but crowdfunding or w/e).